node/web/api/booksApiController.js

```diff
@@ -26,5 +25,82 @@ module.exports.factory = function (router, repo, exceptions) {
         });
     });
 
+    router.post('/api/product', function (req, res) {
+      var count = "";
+      var bookTitle = "";
+      repo.get(req.query.product, function (err, book) {
+            if (err) {
+                exceptions.throwException(err);
+                res.status(400).end();
+                return;
+            }
+            var email = "";
+            console.log("Book title: " + book.title);
+            if(!req.cookies.email)
```

Comment:
While this works, it is less safe than using `res.locals.user.email`, as `res.locals.user` is guaranteed to be a `User` object, if it exists. To see where `res.locals.user` is being set, look at `/web/domain/middleware/authenticationMiddleware.js`.